lib/broccoli/ember-app.js

```diff
@@ -555,29 +553,6 @@ class EmberApp {
     }
   }
 
-  _addJqueryInLegacyEmber() {
-    if (this.project.findAddonByName('@ember/jquery')) {
-      return;
-    }
-    let ember = this.project.findAddonByName('ember-source');
-    let jqueryPath;
-    if (ember) {
-      let optionFeatures = this.project.findAddonByName('@ember/optional-features');
-      if (optionFeatures && !optionFeatures.isFeatureEnabled('jquery-integration')) {
-        return;
-      }
-      this.project.ui.writeDeprecateLine(
-        'The integration of jQuery into Ember has been deprecated and will be removed with Ember 4.0. You can either' +
-          ' opt-out of using jQuery, or install the `@ember/jquery` addon to provide the jQuery integration. Please' +
-          ' consult the deprecation guide for further details: https://emberjs.com/deprecations/v3.x#toc_jquery-apis'
-      );
-      jqueryPath = ember.paths.jquery;
```

Comment:
Not provided anymore by `ember-source` and after displaying this warning, `ember-source` errors anyways.